app-dicts/freedict-deu-eng/freedict-deu-eng-1.0.ebuild

```diff
@@ -1,11 +1,10 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-FORLANG="German"
-TOLANG="English"
 inherit freedict
 
+DESCRIPTION="Freedict for language translation from German to English"
+
```

Comment:
I'd remove that empty line in all ebuilds, because there are only two variables left.
(But of course, this is entirely a matter of personal taste.)